### PR TITLE
doc: add pipeline overview

### PR DESCRIPTION
**Pull request by Xieql**
**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:
part of #493

### CODE REVIEW

File: docs/content/en/docs/pipeline/_index.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+- **Customization of Tasks**: Enables users to tailor their pipelines with both common preset CI tasks and custom tasks, catering to a broad range of needs, enhancing accessibility and adaptability to various skill levels.
+
+- **Enhanced Operational Control and Visibility**: Expands Kurator CLI capabilities, allowing users to list pipelines and view aggregated logs from the actual execution pods.
+
+- **Enhanced Security and Compliance**: Integrates Tekton Chains for automated artifact builds, signature additions, and provenance proof, aligning with Software Supply Chain Level for Software Artifacts (SLSA) standards. It also supports automatic synchronization and uploading of images to repositories.
+
+## Architecture
+
+Below is the architecture diagram of the Kurator Fleet Backup feature.
```
Comment (hzxuzhonghu) [inline docs/content/en/docs/pipeline/_index.md:36]: ?

Comment (Xieql) [reply docs/content/en/docs/pipeline/_index.md]: fixed

**Review by hzxuzhonghu — APPROVED** [review]
/lgtm

**Pull request merged**